package.json

```diff
@@ -239,7 +239,8 @@
   },
   "resolutions": {
     "**/mini-store": "^1.1.0",
-    "**/redux": "3.7.2"
+    "**/redux": "3.7.2",
+    "**/rc-tree": "^5.7.12"
```

Comment:
For some reason, there we several versions, including older ones, being imported by yarn. That leads to typescript errors are necessitates this fix.

See als Slack conversion for reference: https://scm.slack.com/archives/C5AKLAV0B/p1704200386030179

This can likely be remove in antd5.